spec/fixtures/files/text_file.txt

```diff
@@ -0,0 +1 @@
+abcd
```

Comment:
If this file is not being used in testing that's fine, but in this case you should remove it